core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Stack.java

```diff
@@ -798,11 +805,12 @@ public boolean isMaintenanceModeEnabled() {
     }
 
     public boolean isStopInProgress() {
-        return STOP_IN_PROGRESS.equals(getStatus()) || STOP_REQUESTED.equals(getStatus());
+        return STOP_IN_PROGRESS.equals(getStatus()) || STOP_REQUESTED.equals(getStatus()) || EXTERNAL_DATABASE_STOP_IN_PROGRESS.equals(getStatus());
     }
 
     public boolean isReadyForStart() {
-        return STOPPED.equals(getStatus()) || START_REQUESTED.equals(getStatus()) || START_IN_PROGRESS.equals(getStatus());
+        return STOPPED.equals(getStatus()) || START_REQUESTED.equals(getStatus()) || START_IN_PROGRESS.equals(getStatus())
+                || EXTERNAL_DATABASE_STOP_FINISHED.equals(getStatus());
     }
```

Comment:
I think it would make sense to also include `EXTERNAL_DATABASE_START_IN_PROGRESS` and `EXTERNAL_DATABASE_START_FINISHED` here. In particular, this method is called from 2 places:

* `com.sequenceiq.cloudbreak.service.cluster.flow.ClusterOperationService.start()`: Looks okay, the call is wrapped inside a `if (!stack.isStartInProgress()) {}` check that will correctly recognize RDS start statuses as well.
* `com.sequenceiq.cloudbreak.service.stack.flow.StackOperationService.start()` this one will likely fail with a `BadRequestException`, unless the above two statuses are also included here. Actually, that method references the superclass method `com.sequenceiq.cloudbreak.view.StackView.isReadyForStart()`.

Reply:
Let's see :) 
By the time the code gets here the external db is already started. So the `EXTERNAL_DATABASE_STOP_FINISHED` will be added only.